inc/issue.class.php

```diff
@@ -138,99 +172,99 @@ public function display($options = array()) {
+//             } else {
+//                echo '<span class="plugin_formcreator_first"><a href="' .$nav_url. '&stid=' . $firstTicketId . '">'
+//                   .'<img src="' . $CFG_GLPI['root_doc'] . '/pics/first.png" alt="'
+//                   . __('First'). '" title="'
+//                   . __('First'). '" class="pointer"></a></span>';
+//             }
+
+//             if ($previousTicketId == 0) {
+//                echo '<span class="plugin_formcreator_left">'
+//                      .'<img src="' . $CFG_GLPI['root_doc'] . '/pics/left_off.png" alt="'
+//                      . __('First'). '" title="'
+//                      . __('First'). '" class="pointer"></span>';
+//             } else {
+//                echo '<span class="plugin_formcreator_left"><a href="' . $nav_url. '&tid=' . $previousTicketId . '">'
+//                      .'<img src="' . $CFG_GLPI['root_doc'] . '/pics/left.png" alt="'
+//                      . __('Previous'). '" title="'
+//                      . __('Previous'). '" class="pointer"></a></span>';
+//             }
+
+//             if ($lastTicketId == 0) {
+//                echo '<span class="plugin_formcreator_last">'
+//                      .'<img src="' . $CFG_GLPI['root_doc'] . '/pics/last_off.png" alt="'
+//                      . __('First'). '" title="'
+//                      . __('First'). '" class="pointer"></span>';
+//             } else {
+//                echo '<span class="plugin_formcreator_last"><a href="' . $nav_url . '&tid=' . $lastTicketId . '">'
+//                      .'<img src="' . $CFG_GLPI['root_doc'] . '/pics/last.png" alt="'
+//                      . __('Last'). '" title="'
+//                      . __('Last'). '" class="pointer"></a></span>';
+//             }
+
+//             if ($nextTicketId == 0) {
+//                echo '<span class="plugin_formcreator_right">'
+//                      .'<img src="' . $CFG_GLPI['root_doc'] . '/pics/right_off.png" alt="'
+//                            . __('First'). '" title="'
+//                                  . __('First'). '" class="pointer"></span>';
+//                } else {
+//                echo '<span class="plugin_formcreator_right"><a href="' .$nav_url . '&tid=' . $nextTicketId . '">'
+//                      .'<img src="' . $CFG_GLPI['root_doc'] . '/pics/right.png" alt="'
+//                      . __('Next'). '" title="'
+//                      . __('Next'). '" class="pointer"></a></span>';
+//             }
+//             echo '<div class="navigationheader big b">'.
+//                  Html::makeTitle(__('Threads', 'formcreator'), $ticketIndex, count($ticketIds)).
+//                  '</div>';
+
+//             echo '</div>';
+
+//             echo "<div class='timeline_box'>";
+//             $rand = mt_rand();
+//             $ticket->showTimelineForm($rand);
+//             $ticket->showTimeline('123456');
+//             echo "</div>";
+//          }
```

Comment:
I found this became dead code due to your changes. I commented it out to show this fact. A agree to remove them